compose/src/main/java/com/bottlerocketstudios/compose/home/PreviewData.kt

```diff
@@ -0,0 +1,67 @@
+package com.bottlerocketstudios.compose.home
+
+import com.bottlerocketstudios.brarchitecture.domain.models.Link
+import com.bottlerocketstudios.brarchitecture.domain.models.Links
+import com.bottlerocketstudios.brarchitecture.domain.models.Repository
+import com.bottlerocketstudios.brarchitecture.domain.models.User
+import com.bottlerocketstudios.brarchitecture.domain.models.Workspace
+import java.time.ZonedDateTime
+
+val testCard1 = UserRepositoryUiModel(
```

Comment:
@VanyaHuamanBottleRocket Prefix all test properties with `internal` to prevent unintended leakage to other modules.